config/jobs/kubernetes-sigs/cluster-api-provider-cloudstack/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+  - rohityadavcloud
+  - davidjumani
+  - maxdrib
+  - rejoshed
```

Comment:
it seems maxdrib and rejoshed are not member of the kubernetes org

Reply:
I have removed these individuals while they apply for membership to kubernetes, they are currently members of `kubernetes-sigs`.